cliquet/__init__.py

```diff
@@ -30,6 +30,7 @@
     'cache_backend': '',
     'cache_url': '',
     'cache_pool_size': 25,
+    'cache_transaction_request': True,
```

Comment:
I am a bit confused by the granularity of this configuration. it seems to me that it's a recipe for failure to allow transaction on storage but not on cache and/or permissions -- since their data is interdependant. imho, we should use transaction or not.